source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,93 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``*Collection.FindOne()`` method. 
+
+..
+   method
+
+The ``FindOne()`` method takes a query document that specifies the
+subset of your collection that you would like to retrieve a document from. If you
+pass the ``FindOne()`` method an empty document, MongoDB will consider your
+entire collection when retrieving your document. For more information on documents
+in the MongoDB Go driver, see our
+:doc:`guide on data formats </fundamentals/data-formats>`.
+
+You must pass the ``FindOne()`` method a ``context.Context`` interface type
+value. For more information on the ``context.Context`` interface and the
+``context`` package, see our :doc:`guide on context </fundamentals/context>`.
+
+..
+   options
+
+You can specify additional options when retrieving a document such as whether to
+**sort** the matches of your query, which retrieves the matching document
+with the minimum or maximum value as specified in your sort criteria. You can
+also specify a **projection**, which configures the fields in the returned
+document. You can specify additional options by passing the ``FindOne()`` method
+a pointer to an instance of a ``FindOneOptions`` value. You can configure a
+``FindOneOptions`` value by using setter methods such as the
```

Comment:
Thanks for the note. I did some research in the [Donovan and Kernighan Book](https://www.gopl.io/) and Chapter 4 (types chapter which has a section on structs) seems to avoid referring to structs as objects. The methods chapter, chapter 6, is a little more loose with using the term "object", but the chapter seems to lean towards the language "methods of a type" rather than "methods of an object." A tour of go also seems to avoid the use of the term "object", both when discussing [pointers to structs](https://tour.golang.org/moretypes/4) and when discussing [methods](https://tour.golang.org/methods/1). Based off of this research I think we should avoid the term "object" in go documentation, and instead opt for "struct" or "interface type". 

That being said, I agree "value" is the incorrect term here. I am changing it to "a ``FindOneOptions`` struct". Fix: 

In regards to the term "setter", here is an excerpt from the Donovan and Kernighan book:

> Functions that merely access or modify internal values of a type, such as the methods of the Logger type from log package, below, are called getters and setters. However, when naming a getter met ho d, we usually omit t he Get prefix. This preference for brevity extends to al l methods, not just field accessors, and to other redundant prefixes as well, such as Fetch, Find, and Lookup.